📦 Update: Viewport (userScalable, sv)

agamycode opened the pull request:
# Disable viewport zooming on Mobile.

Disables the ability to zoom the viewport on mobile devices.

<img src="https://github.com/user-attachments/assets/e1d3f626-d3d7-40a9-a691-5ecc694ef9d3" alt="userScalable-after" width="300" />
<img src="https://github.com/user-attachments/assets/30d382aa-b1ed-4b46-bf14-91f81b5ef5d1" alt="userScalable-before" width="300" />

### Changes

```typescript
// frontend\src\app\layout.tsx

export const viewport: Viewport = {
  userScalable: false,
};
```
# Update Viewports from `v` to `sv`

updates the viewport units from `v` to `sv` across the project for better responsiveness on small screens.

<img src="https://github.com/user-attachments/assets/65be6065-47eb-47c8-907f-887c6d3f37f4" alt="center-after" width="300" />
<img src="https://github.com/user-attachments/assets/e3d7a6fc-73ea-44b4-aa03-0e219c2449a5" alt="center-before" width="300" />


### Changes

```bash
frontend\src\globals.css
frontend\src\app\layout.tsx
frontend\src\app\pathway\page.tsx

- Replaced `vw` with `svw`.
- Replaced `vh` with `svh`.
```

# Center Children in Layout with `justify-center`

updates the layout to center its children using the `justify-center` class.

## Changes

- Added `justify-center` to the layout's child container to center its children vertically.

agamycode committed:
📦 Update: Viewport (userScalable, sv)

jckli commented:
Hi. I like the changes you made, but is it really necessary to center all children in layout? I feel like if a component needs to be centered, it should just be centered within its own component, not for all children in the body.

The rest look good though and thanks for helping improve my project!

agamycode commented:
Hi, Thank you for your feedback! I'll update the pull request to ensure that only specific components are centered, rather than centering all children in the layout.

agamycode committed:
📦 Update: Viewport (userScalable, sv)

agamycode commented:
I've updated the pull request:

Removed `justify-center` from the layout file.
Added `h-full` to the pathway page to ensure proper height handling.
